chia/cmds/plotnft.py

```diff
@@ -130,8 +122,7 @@ def join_cmd(wallet_rpc_port: Optional[int], fingerprint: int, id: int, fee: int
 
     from .plotnft_funcs import join_pool
 
-    extra_params = {"pool_url": pool_url, "id": id, "fee": fee, "yes": yes}
-    asyncio.run(execute_with_wallet(wallet_rpc_port, fingerprint, extra_params, join_pool))
+    asyncio.run(join_pool(wallet_rpc_port, fingerprint, pool_url, Decimal(fee), id, yes))
```

Comment:
`yes` is a pretty cryptic name

Reply:
i kept the existing name but i could go around and change them.